CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+- Adds `pndev run` to run commands in pndev.toml
+- Adds `pndev gh` to open project repo with current branch
```

Comment:
Proposal: proxy this, and any arguments, to the [`gh`](https://github.com/cli/cli) Github CLI tool.

Reply:
why not add the `gh` tool directly to our nix configs?

that said I see pndev also as a learning ground for us to experiment and learn some rust so I think we should leave ben's work in but yeah I agree that https://cli.github.com/ is pretty great